app/assets/javascripts/application/suggest_user.js

```diff
@@ -0,0 +1,71 @@
+const counterFactory = () => {
+    let cnt = 0;
+    return {
+        count() {
+            return ++cnt;
+        },
+        reset(n) {
+            cnt = 0;
+        },
+        get value() {
+            return cnt;
+        }
+    }
+}
+const counter = counterFactory();
+const suggestions = () => document.getElementById('user_suggestions');
```

Comment:
Please avoid arrow functions, in the current setup it fails like most of the ES6 features.